build/gulp/plugins/util/getComponentInfo.ts

```diff
@@ -111,7 +111,29 @@ const getComponentInfo = (filepath: string, checksum?: string) => {
   // sort props
   info.props = _.sortBy(info.props, 'name')
 
+  // available behaviors
+  info.behaviors = getAvailableBehaviors(info.props.accessibility)
+
   return info
 }
 
+const getAvailableBehaviors = accessibilityProp => {
+  const availableBehaviorNames =
+    (accessibilityProp &&
+      _.get(_.find(accessibilityProp.tags, { title: 'available' }), 'description')) ||
```

Comment:
Nesting is hard to read, may be the code below will be more readable?

```js
const docTags = accessibilityProp.tags

const availableTag= _.find(docTags, { title: 'available' }))
const availableBehaviorNames = _.get(availableTag, 'description', '')
```

As I understand it will also allow us to drop a condition below:

```diff
 return (
--    availableBehaviorNames &&
    availableBehaviorNames
      .replace(/\s/g, '')
      .split(',')
      .map(name => ({
        name,
        displayName: _.upperFirst(name.replace('Behavior', '')),
        category: _.upperFirst(name.split(/(?=[A-Z])/)[0]),
      }))
  )
```